plugins/destination/clickhouse/go.mod

```diff
@@ -3,21 +3,18 @@ module github.com/cloudquery/cloudquery/plugins/destination/clickhouse
 go 1.19
 
 require (
-	github.com/ClickHouse/clickhouse-go/v2 v2.9.2
+	github.com/ClickHouse/clickhouse-go/v2 v2.10.0
 	github.com/apache/arrow/go/v13 v13.0.0-20230601070034-e07e22c5580a
 	github.com/cloudquery/plugin-pb-go v1.0.8
-	github.com/cloudquery/plugin-sdk/v3 v3.5.2
+	github.com/cloudquery/plugin-sdk/v3 v3.8.1
 	github.com/google/uuid v1.3.0
 	github.com/rs/zerolog v1.29.1
-	github.com/stretchr/testify v1.8.2
+	github.com/stretchr/testify v1.8.3
 	golang.org/x/sync v0.2.0
 )
 
-// TODO: remove once al updates are merged
-replace github.com/apache/arrow/go/v13 => github.com/cloudquery/arrow/go/v13 v13.0.0-20230509053643-898a79b1d3c8
```

Comment:
Should we keep this and update the `github.com/cloudquery/arrow` reference? Other plugins still use the fork and it seem we'd always want to use our fork to avoid waiting for changes to be merged upstream.

Anyway this discussion seems out of scope for this PR

Reply:
This might go any other way, as I tend to get the review & merge upstream first & then sync forc, merge `main` into `cqmain` and bump the replace.

I think, for now it might be a place to start the migration to the upstream (and, if there's a long awaiting PR that we really need to propagate faster - we can always add the replace back (as we would still need to bump in anyways))